config-bot/main

```diff
@@ -329,7 +350,6 @@ class Git:
     async def __aenter__(self):
         logging.info("acquiring lock")
         await self._lock.acquire()
-        self.cmd('fetch', 'origin', '--prune', '+refs/heads/*:refs/heads/*')
```

Comment:
I don't have a lot of knowledge of the async code so I'll ask:

This used to happen after "acquiring lock" - does this PR have any implications there?

Reply:
It's still done under lock; it's kept while using the `Git` context.

Reply:
That's a good sanity-check though!